examples/jsm/utils/SkeletonUtils.d.ts

```diff
@@ -28,5 +28,6 @@ export namespace SkeletonUtils {
 
 	export function getEqualsBonesNames( skeleton: Skeleton, targetSkeleton: Skeleton ): string[];
 
-	export function clone( source: Object3D | Skeleton ): Object3D | Skeleton;
+	export function clone( source: Object3D ): Object3D;
+	export function clone( source: Skeleton ): Skeleton;
```

Comment:
I'm afraid this is not correct. If you look at the implementation of `clone()`, you will see that it expects any instance of `Object3D` and only processes objects of type `SkinnedMesh`.